app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+  after_save :increase_posts_counter
+  after_destroy :decrease_posts_counter
+
+  private
+
```

Comment:
- [ ] According to the requirements, the Post model should have a method that returns the 5 most recent comments for a given post. Kindly implement it.

_**Hint:**_ You can implement this the same way you implemented the method `last_posts` in the User model file but it should return  most recent comments anytime it is called. 👍